packaging/templates/mainnet-v1/sentry/sentry/bor/pbss_config.toml

```diff
@@ -141,11 +142,11 @@ gcmode = "archive"
     # [telemetry.influx.tags]
 
 [cache]
-    # cache = 1024
-    gc = 0
-    snapshot = 20
+    cache = 4096
+    # gc = 25
+    # snapshot = 10
     # database = 50
-    trie = 30
+    # trie = 15
```

Comment:
Can you briefly describe why the defaults updated here?

Reply:
reverted